plugins/ruby-foreman-remote-execution/debian/install

```diff
@@ -1,4 +1,4 @@
 foreman_remote_execution.rb           usr/share/foreman/bundler.d
 cache                                 usr/share/foreman/vendor
-foreman_remote_execution              var/lib/foreman/public/assets
+assets                                var/lib/foreman/public
```

Comment:
Looks okay to me:

<pre>
./var/lib/foreman/public/assets
./var/lib/foreman/public/assets/foreman_remote_execution
./var/lib/foreman/public/assets/foreman_remote_execution/manifest.yml
./var/lib/foreman/public/assets/job_invocations-99630200907501afe5bc9b2593ee2bf7.css
./var/lib/foreman/public/assets/job_invocations-99630200907501afe5bc9b2593ee2bf7.css.gz
./var/lib/foreman/public/assets/job_invocations.css
./var/lib/foreman/public/assets/job_invocations.css.gz
./var/lib/foreman/public/assets/template_input-81fd594267126475ae727e179f34b397.js
./var/lib/foreman/public/assets/template_input-81fd594267126475ae727e179f34b397.js.gz
./var/lib/foreman/public/assets/template_input.js
./var/lib/foreman/public/assets/template_input.js.gz
./var/lib/foreman/public/assets/template_invocation-07fde21d2fcad9abc6bacd5089d8649e.css
./var/lib/foreman/public/assets/template_invocation-07fde21d2fcad9abc6bacd5089d8649e.css.gz
./var/lib/foreman/public/assets/template_invocation-3f60f1b8b6ea70f9814826eb94936af4.js
./var/lib/foreman/public/assets/template_invocation-3f60f1b8b6ea70f9814826eb94936af4.js.gz
./var/lib/foreman/public/assets/template_invocation.css
./var/lib/foreman/public/assets/template_invocation.css.gz
./var/lib/foreman/public/assets/template_invocation.js
./var/lib/foreman/public/assets/template_invocation.js.gz
</pre>